content/blog/2016-01-26-one-year-of-open-prometheus-development.md

```diff
@@ -0,0 +1,148 @@
+---
+title: One Year of Open Prometheus Development
+created_at: 2016-01-26
+kind: article
+author_name: Julius Volz
+---
+
+## The beginning
+
+A year ago today, we officially announced Prometheus to the wider world. This
+is a great opportunity for us to look back and share some of the wonderful
+things that have happened to the project since then. But first, let's start at
+the beginning.
+
+Although we already started Prometheus as an open-source project on GitHub in
+2012, we didn't make noise about it at first. We wanted to give the project
+time to mature and be able to experiment without friction. Prometheus was
+gradually introduced for production monitoring at
+[SoundCloud](https://soundcloud.com/) in 2013 and then saw more and more
+usage within the company, as well as some early adoption by our friends at
+Docker and Boexever in 2014. Over the years, Prometheus was growing more and
+more mature and although it was already solving people's monitoring problems,
```

Comment:
Would a comma after 'mature' help parsing (to note that the 2nd "and" connects two sentences into one)? (Or break this long complicated sentence into two?)